jekyll/_cci2/language-javascript-experimental.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: classic-docs
+title: "Configuring a Node.js Application on CircleCI"
+short-title: "JavaScript"
+description: "Building and Testing with JavaScript and Node.js on CircleCI"
+categories: [language-guides]
+order: 5
+version:
+- Cloud
+- Server v3.x
+- Server v2.x
+---
+
+{% raw %}
```

Comment:
Why does this need to be wrapped in `raw` tags?

Reply:
it's a jekyll/liquid bug that I encountered and this fixed it.

Reply:
otherwise it interprets it as a code block oddly.

Reply:
This guy breaks it down a bit more @ccgeller https://ozzieliu.com/2016/04/26/writing-liquid-template-in-markdown-with-jekyll/

Reply:
I'm going to ask about this particular thing again. I was curious so I pulled down the branch and deleted the raw tags and it is fine. What browser/environment is showing it as a code block? I was curious because if this was breaking here it would likely be breaking in other places.

Reply:
@devinmarieb I didn't experience it consistently but only sometimes when it recompiled.

Reply:
@nramabad Curious. Might be good to do some kind of audit to see if other pages have this issue because I imagine if it's happening here it's likely happening on other pages as well 🤷‍♀️